### ISSUE TITLE
Suggestion to add nanopack in conda/anaconda

### ISSUE THREAD

**Issue by NikoLichi** [wdecoster/nanopack#14]
Hi Wouter,

The tools you develop for long-read sequencing are very nice. Thanks for them!

Currently, I am using kyber to compare 2 bam files. but this is in Rust and it has been a little difficult to make it from source.
Since, you managed to add  NanoComp to conda, I wonder why not add the whole Nanopack to conda so all the tools are available?

All the best,
Niko

**Comment by wdecoster** [comment]
Hi Niko,

Most of the separate tools are available on bioconda, but it is possible kyber is not. Always updating nanopack on bioconda, whenever one of the tools get updated, is an unnecessary overhead. But could you not just download a binary from the releases?  

Best,
Wouter

**Comment by NikoLichi** [comment]
Hi Wouter, 
Thanks. I understand it is a little too much to update all the packages constantly... 
Unfortunately, the precompiled Linux library complains with:
```
./kyber: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by ./kyber)
./kyber: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by ./kyber)
./kyber: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by ./kyber)
./kyber: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./kyber)
```
Besides, the Rust compiling is failing due to some dependencies. I am installing them inside a conda environment but still not possible to have Kyber running.

do you have any tips?

**Comment by wdecoster** [comment]
I will provide a musl build, but I am away for a few days

**Comment by NikoLichi** [comment]
Hi Wouter, 
I managed to compile kyber! My first Rust compilation ;)

In a conda environment with all the other nanopack tools available in Anaconda, I set it to be compiled. It failed because of the cmake:
`thread 'main' panicked at ' failed to execute command: No such file or directory (os error 2) is `cmake` not installed?
`
Then, after installing **cmake**, also via conda, it works well.

I ran `cargo test`. Then solved the issue above and then `cargo build --release`.
I'm just leaving this here in case someone else needs it.

All the best,
Niko
